src/user/UserService.ts

```diff
@@ -107,6 +107,18 @@ const deleteUser = async (id: string) => {
   await User.destroy({ where: { id } });
 };
 
+const passwordResetRequest = async (email: string) => {
```

Comment:
will generate randomString and save it to database upon request of passwordResetToken